QueryEngine/CodeGenerator.h

```diff
@@ -406,6 +412,15 @@ class CodeGenerator {
       const std::unordered_map<llvm::Value*, llvm::Value*>&,
       const CompilationOptions&);
 
+  llvm::Function* getArithWithOverflowFunction(const Analyzer::BinOper* bin_oper,
```

Comment:
I would add comments to these two methods to describe their functionality. Also, the `getArithWithOverflowFunction` name confused me a bit, maybe call it `getArithWithOverflowIntrinsic`.

Reply:
Done!